xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt

```diff
@@ -39,7 +39,12 @@ if(OPENGL_FOUND)
                       FrameBufferObject.h)
 endif()
 
-if(OPENGLES_FOUND)
+if(OPENGLES_FOUND AND (CORE_PLATFORM_NAME STREQUAL android OR
+                       CORE_PLATFORM_NAME STREQUAL ios OR
+                       CORE_PLATFORM_NAME STREQUAL gbm OR
+                       CORE_PLATFORM_NAME STREQUAL imx OR
+                       CORE_PLATFORM_NAME STREQUAL mir OR
+                       CORE_PLATFORM_NAME STREQUAL wayland))
```

Comment:
use `CORE_PLATFORM_NAME_LC` instead of `CORE_PLATFORM_NAME`
see comment at https://github.com/xbmc/xbmc/blob/4dfc7a8/cmake/scripts/common/Platform.cmake#L13-L16

Reply:
ah right, thanks